src/app/loadout/LoadoutView.tsx

```diff
@@ -101,8 +106,16 @@ export default function LoadoutView({
   const categories = _.groupBy(items.concat(warnitems), (li) => li.item.bucket.sort);
   const power = loadoutPower(store, categories);
 
+  const onLoadoutSelect = () => dispatch(streamDeckSelectLoadout(loadout, store));
+
+  const selectable = streamDeckSelection === 'loadout' && !hideShowModPlacements;
+
   return (
-    <div className={styles.loadout} id={loadout.id}>
+    <div
+      className={`${styles.loadout} ${selectable ? styles.sdSelectable : ''}`}
+      id={loadout.id}
+      onClick={onLoadoutSelect}
+    >
```

Comment:
Maybe a nicer way would be to add a button to the loadout for selecting it?

Reply:
Yeah while the selection is active we can show a temporary button like "send to stream deck" or something like that